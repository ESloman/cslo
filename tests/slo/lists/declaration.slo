var x = [1, 2, 3];
println(x);

var y = [];
println(y);
