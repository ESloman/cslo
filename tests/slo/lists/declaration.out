list[3]: [1, 2, 3]
list[0]: []
